docker-compose.yml

```diff
@@ -248,6 +248,7 @@ services:
       cache_from:
         - ${REPO}:${ARCH}-ubuntu-${UBUNTU}-cpp
       args:
+        arch: ${ARCH}
```

Comment:
We should add this to the cuda image for consistency, although I don't think nvidia ships arm images.

Reply:
Good catch!